benches/payout-stakers/src/main.rs

```diff
@@ -21,7 +25,7 @@ use rand::{thread_rng, Rng};
 use sp_keyring::AccountKeyring;
 
 // testcase parameters
-const NOMINATOR_COUNT: u32 = MAX_NOMINATORS_REWARDED_PER_VALIDATOR;
+const NOMINATOR_COUNT: u32 = 2 * MAX_NOMINATORS_REWARDED_PER_VALIDATOR;
```

Comment:
Perhaps a comment as to why the `2 *` would be nice. Are we testing that some nominators don't get any rewards?

Reply:
I added a comment. This particular bench fails when run on r-13.4 version, but it is OK, as those changes prepare this crate to be run in the CI, which is based on `main`, that has paged exposure logic already. In the paged exposure world, all nominators are paid, regardless of how many of them are. In subsequent PR we should add a nighty workflow that runs this bench against `aleph-node` from `main`, and modify this file to call `payout_stalers` in a loop, until `Error::AlreadyClaimed` is received.